test/Conversion/tritongpu_to_llvm.mlir

```diff
@@ -714,3 +714,63 @@ module attributes {"triton_gpu.num-warps" = 1 : i32} {
     return
   }
 }
+
```

Comment:
nit: better to have less ops in the test input.

Reply:
Yeah, I've removed unnecessary logic here.